unpaddedbase64.py

```diff
@@ -15,8 +15,6 @@
 
 import base64
 
-__version__ = "1.1.0"
-
```

Comment:
Is this now autogenerated or could a project possibly relying on this now fail?

Reply:
In the initial move to declarative setuptools, I moved the version into the `metadata.version` key in `setup.cfg`.

With Poetry, the version number is stored in `pyproject.toml` and reflected into all the right places in the wheel / sdist metadata. Anyone depending on that specific attribute existing might break, but... no one should be doing that.